CONTRIBUTING.md

```diff
@@ -122,7 +128,6 @@ You can install them into your current Conda environment with:
 
 ```shell
 conda install -c conda-forge --file requirements-conda.txt
-python -m pip install -r requirements.txt
```

Comment:
Is there a reason this was removed, or was this just an accident? This is still required...

Reply:
```suggestion
python -m pip install -r requirements.txt
```